mplc/scenario.py

```diff
@@ -349,6 +349,14 @@ def __init__(
         if not is_dry_run:
             self.save_folder.mkdir(parents=True, exist_ok=True)
 
+        # ------------------------------------------------------------------
+        # Select in the kwargs the parameters to be transferred to sub object
+        # ------------------------------------------------------------------
+        self.mpl_kwargs = {}
+        for key, value in kwargs.items():
+            if key.startswith('mpl_'):
+                self.mpl_kwargs[key.replace('mpl_', '')] = value
```

Comment:
Are you sure renaming the arguments (by removing `mpl_`) is needed and bring more advantages than risks of future errors?

Reply:
the name is really important as we will use this dict as kwargs, with **, and I think that the keyword arguments of mpl should not be all prefixed by mpl_. Moreover I don't think there are risks, because all these renamed parameters are in a specific kwargs. To be honest I am not really a fan of this solution, but I did not find a better one.

Reply:
OK. So if I understand it correctly: when initializing a scenario, if a user wants to pass kwargs to the mpl that will be instantiated by the scenario, she should prefix them by `mpl_`. Right?
I could work with that. But it implies a very clear documentation for users right?